modules/sdk-coin-eos/src/eos.ts

```diff
@@ -360,11 +360,11 @@ export class Eos extends BaseCoin {
    * Convert a currency amount represented in base units (satoshi, wei, atoms, drops, stroops)
    * to big units (btc, eth, xrp, xlm)
    */
-  baseUnitsToBigUnits(baseUnits: string | number): string {
-    const dividend = this.getBaseFactor();
+  baseUnitsToBigUnits(baseUnits: string | number, chex?: boolean): string {
+    const dividend = chex ? 1e8 : this.getBaseFactor();
     const bigNumber = new BigNumber(baseUnits).dividedBy(dividend);
     // set the format so commas aren't added to large coin amounts
-    return bigNumber.toFormat(4, null as any, { groupSeparator: '', decimalSeparator: '.' });
+    return bigNumber.toFormat(chex ? 8 : 4, null as any, { groupSeparator: '', decimalSeparator: '.' });
```

Comment:
we should only need to change the hard coded `4` to `this.decimalPlaces`.

we define decimal places in statics: https://github.com/BitGo/BitGoJS/blob/master/modules/statics/src/coins.ts#L1750-L1757